js/app.js

```diff
@@ -1,23 +1,92 @@
+  priceProperty,
+  pricePerM,
+  taxPCC,
+  sourcingCost,
+  electricityCost,
+  heatingCost,
+  interiorProjectCost,
+  homestagingCost,
+  sellingCost,
+  insuranceCost,
+  allCosts;
 const btnInvestVariables = document.getElementById("btnInvestVariables");
-let investTime, propertySize, pricePerM, priceProperty, taxPCC, sourcingCost;
 
-const COURT_COST = 779;
-const NOTARY_COST = 1200;
+const COURT_COST = 779; //per deal
+const NOTARY_COST = 1200; //per deal
+const SOURCING_COST = 5000; //per deal
+const ELECTRICITY_COST = 50; //per month
+const WATER_COST = 20; //per month
+const HEATING_COST = 0; //in summer per month
+const INTERIOR_PROJECT_COST = 800; //per deal
+const HOMESTAGING_COST = 300; //per deal
+const SELLING_COST = 200; //per deal
+const PROPERTY_INSURANCE_COST = 350; //per deal
 
 function setInvestmentVariables() {
   investTime = parseInt(document.getElementById("investTime").value);
   propertySize = parseInt(document.getElementById("propertySize").value);
   priceProperty = parseInt(document.getElementById("priceProperty").value);
-  setVariableRelatedValues();
+  setVariableRelatedCosts();
+  renderVariableRelatedCosts();
+  setOtherCosts();
+  renderOtherCosts();
+  addAllCosts();
+  renderAllCosts();
 }
-
-function setVariableRelatedValues() {
+function setVariableRelatedCosts() {
   pricePerM = priceProperty / propertySize;
-  document.querySelector("#pricePerM > span").innerHTML = pricePerM.toFixed(1);
-  document.querySelector("#courtCosts > span").innerHTML = COURT_COST;
   taxPCC = priceProperty * 0.02;
+  courtCost = COURT_COST;
+  notaryCost = NOTARY_COST;
+}
+
+function renderVariableRelatedCosts() {
+  document.querySelector("#pricePerM > span").innerHTML = pricePerM.toFixed(1);
+  document.querySelector("#courtCosts > span").innerHTML = courtCost;
   document.querySelector("#taxPCC > span").innerHTML = taxPCC;
-  document.querySelector("#notaryCost > span").innerHTML = NOTARY_COST;
+  document.querySelector("#notaryCost > span").innerHTML = notaryCost;
+}
+
+function setOtherCosts() {
+  electricityCost = ELECTRICITY_COST * investTime;
+  waterCost = WATER_COST * investTime;
+  heatingCost = HEATING_COST * investTime;
+  interiorProjectCost = INTERIOR_PROJECT_COST;
+  homestagingCost = HOMESTAGING_COST;
+  sellingCost = SELLING_COST;
+  insuranceCost = PROPERTY_INSURANCE_COST;
+}
+
+function renderOtherCosts() {
+  document.querySelector("#electricityCost > span").innerHTML = electricityCost;
+  document.querySelector("#waterCost > span").innerHTML = waterCost;
+  document.querySelector("#heatingCost > span").innerHTML = heatingCost;
+  document.querySelector("#interiorProjectCost > span").innerHTML = interiorProjectCost;
+  document.querySelector("#homestagingCost > span").innerHTML = homestagingCost;
+  document.querySelector("#sellingCost > span").innerHTML = sellingCost;
+  document.querySelector("#insuranceCost > span").innerHTML = insuranceCost;
+}
+
+function addAllCosts() {
+  allCosts =
```

Comment:
Then you will be able to create this function only once, if you will add new property you will not be forced to change the calculations. 

```suggestion
  allCosts =
    Object.values(costs.monthly).reduce((acc, p) => acc + p * investTimeInMonths, 0) + 
    Object.values(costs.deal).reduce((acc, p) => acc + p, 0);
```